### PR TITLE
fix: Incorrect calculation of resource pack installation progress

### LINKED ISSUE #55: Incorrect calculation of resource pack installation progress
[Issuehunt badges](https://oss.issuehunt.io/r/zardoy/prismarine-web-client/issues/55)

Снимок экрана 2024-01-13 в 21 06 44

https://mcraft.fun/?texturepack=https%3A%2F%2Fzardoy.site%2Fwp-content%2F%3Ffile%3Duploads%2F2023%2F09%2FPollachiusv1.0.0for1.8.zip

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $10.00)

- [zardoy zardoy](https://oss.issuehunt.io/u/zardoy) ($10.00)

### Submitted pull Requests
- [#74 fix: issue #55: Incorrect calculation of resource pack installation progress](https://oss.issuehunt.io/r/zardoy/prismarine-web-client/pull/74)
---

### Tips

- Checkout the [Issuehunt explorer](https://oss.issuehunt.io/r/zardoy/prismarine-web-client/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://oss.issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

### PR DESCRIPTION
**Pull request by AliLordLoss**
resolves #55

### CODE REVIEW

File: src/texturePack.ts [hunk 1]
```diff
@@ -83,7 +83,6 @@ export const installTexturePack = async (file: File | ArrayBuffer, name = file['
     if (path.endsWith('/')) return
     await mkdirRecursive(dirname(writePath))
     await fs.promises.writeFile(writePath, Buffer.from(await file.async('arraybuffer')))
-    done++
```
Comment (zardoy) [inline src/texturePack.ts:86]: no way it was that simple, but let's remove ++done in function instead. let's show for others as its better to have side-effect free functions, okay?

**Review by zardoy — CHANGES_REQUESTED** [review]

File: src/texturePack.ts (hunk 1 above)
Comment (AliLordLoss) [reply src/texturePack.ts:86]: Yeah it was that simple :)
Roger that, I'll make the change

Comment (zardoy) [reply src/texturePack.ts]: Thank you a lot!

**Review by zardoy — APPROVED** [review]

**Comment by zardoy** [comment]
I updated the base branch to an active one (next). The change is deployed to s.mcraft.fun for now, I like to make new releases when there are enough features and fixes to release!
UPD: Thank you so much for the contribution!

**Comment by AliLordLoss** [comment]
You're welcome! And thank you too :)

**Comment by zardoy** [comment]
I'm sorry for changing the base branch. I also forgot GitHub doesn't close issues until the fix is merged into the main branch which is very annoying for me.
I think I'm just going to change the default branch of the repository to ensure it won't happen again!

**Comment by AliLordLoss** [comment]
Oh it's alright, no worries.

**Pull request merged**